app/models/provider_relationship_permissions.rb

```diff
@@ -4,7 +4,7 @@ class ProviderRelationshipPermissions < ApplicationRecord
 
   PERMISSIONS = %i[make_decisions view_safeguarding_information view_diversity_information].freeze
 
-  validate :at_least_one_active_permission_in_pair, if: -> { setup_at.present? }
+  validate :at_least_one_active_permission_in_pair, if: -> { setup_at.present? || validation_context == :setup }
```

Comment:
I think we can completely remove the `if` condition here, for the new implementation we will always use the model to validate regardless of `setup_at` being `nil` or not. In the old implementation we have the wizard validating records with `setup_at` not present so not sure this is required anymore. Let me know your thoughts.

Reply:
Yeah, when the permissions are created initially all of the values are set to `false`. We wouldn't be able to save them without that condition.

Reply:
I think the current implementation with `validation_context` is ok, as we do need the `setup_at` check as well for the TTAPI sync 👍